crates/chain/src/keychain/txout_index.rs

```diff
@@ -206,22 +190,18 @@ const DEFAULT_LOOKAHEAD: u32 = 25;
 /// [`unused_spks`]: KeychainTxOutIndex::unused_spks
 #[derive(Clone, Debug)]
 pub struct KeychainTxOutIndex<K> {
-    inner: SpkTxOutIndex<(DescriptorId, u32)>,
+    inner: SpkTxOutIndex<(K, u32)>,
     // keychain -> (descriptor, descriptor id) map
```

Comment:
🔧  needs to match the code change.
```suggestion
    // keychain -> descriptor id map
```